compiler/src/compiler_test.go

```diff
@@ -2,10 +2,19 @@ package compiler
 
 import (
 	"fmt"
+	"github.com/rs/zerolog"
+	"io/ioutil"
 	"os/exec"
 	"testing"
 )
 
+func initLogger() zerolog.Logger {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+	consoleWriter := zerolog.ConsoleWriter{Out: ioutil.Discard}
+	logger := zerolog.New(consoleWriter)
+	return logger
+}
+
```

Comment:
The extraction of this dummy logger to a unified file also seems like a good idea.

Reply:
done